api/envoy/api/v2/core/health_check.proto

```diff
@@ -21,10 +21,16 @@ option (gogoproto.equal_all) = true;
 message HealthCheck {
   // The time to wait for a health check response. If the timeout is reached the
   // health check attempt will be considered a failure.
-  google.protobuf.Duration timeout = 1 [(validate.rules).duration.required = true];
+  google.protobuf.Duration timeout = 1 [
+    (validate.rules).duration = {required: true, gt: {}},
```

Comment:
nit: IMO saying greater than 0 is a bit more clear here. Same below.

Reply:
I thought the same, but then I found that the others use `gt: {}` only. E.g. https://github.com/envoyproxy/envoy/blob/5d2bd6e0f16167db1f57282525384d0efb140aff/api/envoy/config/filter/http/buffer/v2/buffer.proto#L23

Reply:
mmm I still think that having the explicit value is more clear. I would rather add it here and update the other protos in a separate PR for explicitness.

Reply:
@mattklein123 @junr03 will do. I also think that we should automatically render that requirement into doc, e.g. REQUIRED, GREATER THAN 0 SEC. But I believe that should be down in a different PR. Related: https://github.com/envoyproxy/envoy/issues/3731

Reply:
@junr03 done in 1d2b066.